### ISSUE TITLE
Announcement of new core members for second semester of 2024

### ISSUE THREAD

**Issue by mikelkou** [scverse/governance#73]
Core announcements: Emma (?), Severin, Tim, Wouter

**Comment by grst** [comment]
What about Can, Ori, Ilan, Tim (?)

**Comment by mikelkou** [comment]
Yeap! You are right. Probably none of them has been announced. I will make a big announcement with the 2024 acquisitions 😅 But first I want to see where I stopped announcing. Horrible.

**Comment by Zethson** [comment]
@mikelkou what's the status here, please?

**Comment by Zethson** [comment]
@mikelkou ?

**Comment by mikelkou** [comment]
No status here. We are not doing it anymore unfortunately. It's never late to embrace all the "new" members if we want to. Like, talk about how much we grew and mention all the members that joined the last year or so.

**Comment by Zethson** [comment]
Yeah, would you be willing to do that, please? Or ask Kartik or someone else?

**Comment by mikelkou** [comment]
No I can do it. Too personal for Kartik. I will combine it with a summer break announcement. :) Thanks for the reminder.

**Comment by mikelkou** [comment]
@timtreis Can you add it on the shameboard? Deadline end of July

**Comment by Zethson** [comment]
May or may not have happened but it's been a year.
